yijing/iching_export.md

```diff
@@ -257,7 +257,7 @@
 {\Large 亨。匪我求童蒙，童蒙求我。初筮告，再三瀆，瀆則不告。利貞}
```

Comment:
I think we should remove all of the punctuation in the Chinese. I was learning that this is really 20th-century style ... the punctuation may be the biggest fountain of gloss in the whole corpus ... maybe the meaning becomes even more broad without the artificial divisions? Are they artificial? Hrm ...